src/templates/search-license-dropdown.html

```diff
@@ -0,0 +1,35 @@
+<!-- 
+    Copyright (C) 2018 Gopalakrishnan
+
+    SPDX-License-Identifier: GPL-3.0-or-later
+    See GPL-3.0-or-later in the Licenses folder for license information
+-->
+
+<style>
+.ui.search.dropdown.selection {
+    float: right;
+    margin-top: -30px;
+}
+.ui.error.message, .ui.success.message {
+    margin-bottom: 50px;
+}
+</style>
+
+<script>
+window.onload = function () {
+    var select_license = document.getElementById('select-license');
+    select_license.onchange = function () {
+	var id = select_license.options[select_license.selectedIndex].value;
+	window.location.href = '/license/' + id;
```

Comment:
why move to a license page?

Reply:
The License information has been maintained in the `/license/<int:id>`. Hence on such a redirect, the complete information of the license can be obtained.